### PR TITLE
fixed scale bug.

### PR DESCRIPTION
**Pull request by michaelthatsit**

### CODE REVIEW

**Comment by michaelthatsit** [comment]
@hanbollar I added a global directly to mrjsUtils so lemme know if you think there'd be a better place for it, I also have other globals we could probably coral under mrjsUtils or maybe create mrjsEnums?

@adnanwahab there's a slight change that will impact the TextSystem so you'll wanna merge in my changes when this hits main.

File: src/utils/index.js [hunk 1]
```diff
@@ -30,6 +30,13 @@ const mrjsUtils = {
     Physics,
     StringUtils,
     xr,
+
+    get appScale() {
```
Comment (hanbollar) [inline src/utils/index.js:34]: this should be a function within mr-app even if it is not object dependent - there should be a way to do a `static` function of the class in javascript for it

**Review by hanbollar — APPROVED** [review]
> I added a global directly to mrjsUtils so lemme know if you think there'd be a better place for it, I also have other globals we could probably coral under mrjsUtils or maybe create mrjsEnums

@michaelthatsit for the above - i think it would be useful for us to have one `enums.js` at the same level as `index.js` and `globals.js` at the root of the project, i dont think having it as a completely separate subproject like mrjsUtils is useful

-- also see my other comment about the one function you had added to index.js

otherwise lgtm 🥳

File: src/utils/index.js (hunk 1 above)
Comment (michaelthatsit) [reply src/utils/index.js:34]: I thought about this but not everything has access to `mr-app`. and I'd like to keep it that way, as it'd make things harrier.

Comment (hanbollar) [reply src/utils/index.js:34]: gotcha, then doing something in like `utils/app.js` should be fine (or whatever js you want to call it in `utils/...`) - we just shouldnt directly put a function in the index.js is all for better scalability and future testing

Comment (michaelthatsit) [reply src/utils/index.js:34]: Yeah that's a fair point. I'm cooking and haven't tested your PR yet. I'll make the change after finishing both of those.

Comment (hanbollar) [reply src/utils/index.js:34]: yea no rush ~

**Pull request merged**